src/jdk.jdi/share/classes/com/sun/jdi/ObjectReference.java

```diff
@@ -328,7 +328,7 @@ Value invokeMethod(ThreadReference thread, Method method,
      * @return <code>true</code> if this {@link ObjectReference} has been collected;
      * <code>false</code> otherwise.
      * @throws VMCannotBeModifiedException if the VirtualMachine is read-only
-     * -see {@link VirtualMachine#canBeModified()}.
+     * - see {@link VirtualMachine#canBeModified()}.
```

Comment:
The alternative would be drop the "- see <link>" and add  `@see VirtualMachine#canBeModified()` on a new line so that it shows up in the "See also" list.

Reply:
There are a lot of places where we use `see` in this manner, because it applies specifically to the text that proceeds it. These 3 were different in that the space was missing.

Reply:
I personally would not use this style, instead I would limit the `@throws` to just the simple sentence and then use `@see` for related methods, just as we do in the core APIs.

Reply:
We have 33 instances of using `see` like this in JDI. I rather not get into changing them for this PR.